src/docfx/serve/lsp/credentialRefresh/CredentialRefreshParams.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading.Tasks;
+using MediatR;
+using OmniSharp.Extensions.JsonRpc;
+
+namespace Microsoft.Docs.Build
+{
+    [Parallel]
+    [Method("docfx/userCredentialRefresh", Direction.ServerToClient)]
+    public record CredentialRefreshParams : IRequest<CredentialRefreshResponse>
```

Comment:
use `record` only when you need _value equality semantic_
```suggestion
    public class CredentialRefreshParams : IRequest<CredentialRefreshResponse>
```